pkg/vulnsrc/ubuntu/ubuntu.go

```diff
@@ -52,10 +52,16 @@ var (
 		"noble":    "24.04",
 		"oracular": "24.10",
 		// ESM versions:
-		"precise/esm":      "12.04-ESM",
-		"trusty/esm":       "14.04-ESM",
+		"precise/esm": "12.04-ESM",
+		"trusty/esm":  "14.04-ESM",
+		// Possible multiple values for one release:
+		// (release_list="trusty trusty/esm xenial esm-infra/xenial esm-apps/xenial bionic esm-infra/bionic esm-apps/bionic focal esm-apps/focal jammy esm-apps/jammy noble oracular plucky")
+		// cf. https://wiki.ubuntu.com/SecurityTeam/BuildEnvironment#line867
 		"esm-infra/xenial": "16.04-ESM",
+		"esm-apps/xenial":  "16.04-ESM",
 		"esm-infra/bionic": "18.04-ESM",
+		"esm-apps/bionic":  "18.04-ESM",
+		"esm-apps/focal":   "20.04-ESM",
```

Comment:
If I understand correctly, ESM has two streams, apps and infra.
https://ubuntu.com/security/esm#:~:text='esm%2Dapps'%20and%20',are%20compliant%20with%20FIPS%20regulations.

So, should we also add `esm-infra/focal`?

Reply:
I was just thinking that maybe we'll have to split these sources (like fips, ksplice for Oracle Linux).
But in any case, we need to wait for user requests first.

Reply:
>But Ubuntu just started supporting 20.04-ESM, so there is a chance they will add it.

Yes, that's also my guess. The advisory has not yet been issued for the packages within the esm-infra repository, but I believe it will be published soon.

Reply:
>I was just thinking that maybe we'll have to split these sources (like fips, ksplice for Oracle Linux).

I also found that Ubuntu has fips, like Oracle. We probably need a special handling for them.